Snippets/frontend.sql

```diff
@@ -66,5 +66,6 @@ ON DUPLICATE KEY UPDATE
   `namespace` = VALUES(`namespace`),
   `shopID` = VALUES(`shopID`),
   `name` = VALUES(`name`),
-  `localeID` = VALUES(`localeID`)
+  `localeID` = VALUES(`localeID`),
+  `value` = VALUES(`value`)
```

Comment:
I don't know if this is a problem. Users could overwrite our snippets. With this new Update we would overwrite it back to our snippets every time the plugin is installed (updated)

Reply:
weil der Rest der Datei ja ein ähnliches Problem hat, find ich das zwar Schade, aber nicht soooo schlimm. Hat es einen Grund warum Connect die Snippets nicht in den ini-Files hat?